src/index.html

```diff
@@ -21,7 +24,413 @@
+  <body class="page__body">
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__link"
+      >
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__product-image"
+          src="./images/imac.jpeg"
+          alt="iMac image"
+        />
+
+        <h2 class="card__product-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="card__rating-container">
+          <span class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </span>
+
+          <span class="card__reviews-count">Reviews: 5</span>
+        </div>
+
+        <div class="card__price-container">
+          <span class="card__price-label">Price:</span>
+
+          <span class="card__price-amount">$2,199</span>
+        </div>
+
+        <a
+          href="#"
+          class="card__buy-link"
+          data-qa="card-hover"
```

Comment:
The 'data-qa' attribute is used for testing purposes and should not be included in the production code unless it's specifically being used for testing.